package/autoyast2.spec

```diff
@@ -153,33 +156,6 @@ for d in %{buildroot}%{_datadir}/autoinstall/modules/*.desktop ; do
     %suse_update_desktop_file $d
 done
 
-# Class conf
-install -d -m 700 %{buildroot}%{_sysconfdir}/autoinstall
-# Installation files
-install -d -m 700 %{buildroot}%{_localstatedir}/adm/autoinstall/scripts
```

Comment:
Why are we removing these directories?

Reply:
it is just moved to https://github.com/yast/yast-autoinstallation/pull/529/files#diff-52c976fc38ed2b4e3b1192f8a8e24cffR17